src/containers/samples.json

```diff
@@ -0,0 +1,303 @@
+    "sysCode": "",
+    "samples": [],
+    "result": []
+  },
+  {
+    "lang": "Bengali",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "un-ben-Beng-Latn-2016",
+    "samples": ["",""],
+    "result": []
+  },
+  {
+    "lang": "Bulgarian",
+    "isoName": "UN 1977, III/10",
+    "sysCode": "",
+    "samples": ["Република България"],
+    "result": []
+  },
+  {
+    "lang": "Burmese",
+    "isoName": "BGN/PCGN 1970",
+    "sysCode": "",
+    "samples": ["ြမန်မာ","ရန်ကုန်"],
+    "result": []
+  },
+  {
+    "lang": "Byelorussian",
+    "isoName": "national 2007",
+    "sysCode": "by-bel-Cyrl-Latn-2007",
+    "samples": ["Беларусь","Минск"],
+    "result": []
+  },
+  {
+    "lang": "Chinese",
+    "isoName": "UN 1977, III/8",
+    "sysCode": "acadsin-zho-Hani-Latn-2002",
+    "samples": ["中国","北京"],
+    "result": []
+  },
+  {
+    "lang": "Dzongkha",
+    "isoName": "national 1997",
+    "sysCode": "",
+    "samples": ["འབྲུག་ཡུལ","ཐིམ་ཕུ"],
+    "result": []
+  },
+  {
+    "lang": "Georgian",
+    "isoName": "national 2002",
+    "sysCode": "ggg-kat-Geor-Latn-2002",
+    "samples": ["საქართველო","თბილისი"],
+    "result": []
+  },
+  {
+    "lang": "Greek",
+    "isoName": "UN 1987, V/19",
+    "sysCode": "un-ell-Grek-Latn-1987-tl",
+    "samples": ["Ελλάδα","Αθήνα"],
+    "result": []
+  },
+  {
+    "lang": "Gujarati",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
+    "samples": ["જરાત","ગાંધીનગર"],
+    "result": []
+  },
+  {
+    "lang": "Hebrew",
+    "isoName": "UN 1977, III/13; 2007, IX/9",
+    "sysCode": "icao-heb-Hebr-Latn-9303",
+    "samples": ["מְ דִ ינַת י ִׂשְ ָראֵ ל"],
+    "result": []
+  },
+  {
+    "lang": "Hindi",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "un-hin-Deva-Latn-2016",
+    "samples": [],
+    "result": []
+  },
+  {
+    "lang": "Japanese",
+    "isoName": "national: Kunrei-siki/Mod. Hepburn",
+    "sysCode": "mext-jpn-Hrkt-Latn-1954",
+    "samples": ["日本","東京"],
+    "result": []
+  },
+  {
+    "lang": "Kannada",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "bis-knd-Knda-Latn-13194-1991",
+    "samples": [],
+    "result": []
+  },
+  {
+    "lang": "Kazakh",
+    "isoName": "BGN/PCGN 1979",
+    "sysCode": "odni-kaz-Cyrl-Latn-2015",
+    "samples": ["Қазақстан","Астана"],
+    "result": []
+  },
+  {
+    "lang": "Khmer",
+    "isoName": "UN 1972, II/10",
+    "sysCode": "",
+    "samples": ["កមពុជ","ភនំេពញ"],
+    "result": []
+  },
+  {
+    "lang": "Kirghiz",
+    "isoName": "BGN/PCGN 1979",
+    "sysCode": "odni-kir-Cyrl-Latn-2015",
+    "samples": ["Кыргызстан","Бишкек"],
+    "result": []
+  },
+  {
+    "lang": "Korean",
+    "isoName": "ISO/TR 11941:1996",
+    "sysCode": "iso-kor-Hang-Latn-1996-method1",
+    "samples": ["평양","서울"],
+    "result": []
+  },
+  {
+    "lang": "Korean",
+    "isoName": "ISO/TR 11941:1996",
+    "sysCode": "iso-kor-Hang-Latn-1996-method1",
```

Comment:
`method2`?

Reply:
method1 is for North Korea system and method2 is for South Korea. I'm wondering which one should it be used for transliteration of examples. I used methoed1

Reply:
Pyongyang should use method1 and Seoul should use method2. Can we do that?